src/util/motionModel.ts

```diff
@@ -306,7 +364,10 @@ export const getNextGnss = (): Promise<GnssMetadata[][]> => {
     try {
       console.log('Last file is ' + prevGnssFile);
       pathToGpsFile = await getNextGnssName();
+      copyFileToUSB(pathToGpsFile, USB_WRITE_PATH, FILETYPE.GNSS);
```

Comment:
@HariHive can we do it only if `pathToGpsFile !== prevGnssFile`,
we'll not be copying the same gps file twice then